packages/forms/resources/views/components/textarea.blade.php

```diff
@@ -13,6 +13,8 @@
             ax-load-src="{{ \Filament\Support\Facades\FilamentAsset::getAlpineComponentSrc('textarea', 'filament/forms') }}"
             x-data="textareaFormComponent({ initialHeight: @js($initialHeight) })"
             x-ignore
+            x-on:load.window="render()"
```

Comment:
Shouldn't we just use `x-init` instead?

Reply:
I confirm that `x-init="render()"` has the same behavior as `x-on:load.window="render()"`

Reply:
Should probably even be moved into `init()` in the JS file.

Reply:
This won't work because the `init()` method can only be used in `x-data` or `Alpine.data()` as stated [here](https://alpinejs.dev/directives/init#auto-evaluate-init-method)

There's still one problem where if the `Textarea` is inside a modal, the auto size will work the first time, but not the second time. I think that's due to `x-init` only running once.

One possible but not optimal fix is to dispatch an event in `packages/support/resources/views/components/modal/index.blade.php` inside the `open` method like so:
`setTimeout(() => $wire.dispatch('modal-opened'), 500)` and then listen to it in `packages/forms/resources/views/components/textarea.blade.php` like so: `x-on:modal-opened.window="render()"`

I should note that there's already an `modal-opened` event being fired inside the `open` method:
```javascript
this.$refs.modalContainer.dispatchEvent(
                new CustomEvent('modal-opened', { id: '{{ $id }}' }),
            )
``` 
But it did not work, even after putting it inside `setTimeout`

Reply:
> This won't work because the init() method can only be used in x-data or Alpine.data() as stated [here](https://alpinejs.dev/directives/init#auto-evaluate-init-method)

It should work; we use that pattern in many other components.

Reply:
I did not know that you could use object syntax with `init`. It works btw